src/html/slick-slider.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>slick-slider</title>
+  <link rel="stylesheet" href="../../public/styles/style.css">
+</head>
+<body>
+  <div class="slider-partners">
```

Comment:
create please .container in 000_main.scss
for example: 
.container {
  max-width: $MAX_WIDTH;
  width: 100%;
  padding: 0 15px;
  margin: 0 auto;
}

max-width = 15px padding+ width in psd + 15px padding